### PR TITLE
Add Contributing.md

### PR DESCRIPTION
**Pull request by muffinresearch**
Add contributing markdown file (based on the addons-server one with a few small changes)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Contributing
+============
+
+If you’re not sure this is the correct place to file an issue then please file an issue on the [mozilla/addons] project instead.
```
Comment (tofumatt) [inline CONTRIBUTING.md]: > If you’re not sure this is the correct place to file an issue then please file an issue on the [mozilla/addons] project instead.

Could use with some more context. The first paragraph should provide more info:

```markdown
This is the code for the Mozilla Add-ons website frontend–it's the code that displays Add-ons to user from the Add-ons (API) Server. This project handles the display of data and submission of things like reviews to the API, but doesn't handle the data. If you’re not sure this is the correct place to file an issue then please file an issue on the [mozilla/addons] project instead.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Before contributing code, please note:
```
Comment (tofumatt) [inline CONTRIBUTING.md]: "Before contributing" should suffice, I'd remove the word "code".

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+-   You agree to license your contributions under the [license].
```
Comment (tofumatt) [inline CONTRIBUTING.md]: I thought we mostly used `*` for lists, but it's just a nit.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+-   Please ask on the [dev-addons mailing list] before submitting pull-requests for new features or large changes that are not related to existing issues.
```
Comment (tofumatt) [inline CONTRIBUTING.md]: I would say "Please file issues or ask on the dev-addons list". No reason the discussion can't happen on here.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+Ready to get started? Follow [these steps].
+
+Thank you for contributing!
+
+## Note to staff
+
+If you come across a potential `good first bug` for contributors, please tag it with `maybe good first bug`. The community team [triages] these every other week to ensure they have mentors assigned, onboarding information, and basic steps to get started. This gives new contributors a better experience when they pick a “good first bug” to work on.
+
+
```
Comment (tofumatt) [inline CONTRIBUTING.md]: Could remove a few of these empty lines.

File: CONTRIBUTING.md (hunk 3 above)
Comment (muffinresearch) [inline CONTRIBUTING.md]: Ah that's pandoc's fault :) - I converted this from rst here: https://github.com/mozilla/addons-server/blob/master/CONTRIBUTING.rst

**Pull request merged**